lib/diagnostic.rb

```diff
@@ -37,43 +37,46 @@
 # of multiplying by two the result of adding `a` and `b`. Name the method
 # conventionally.
 
-def method_name
+def mutiply_sum_by_2(a,b)
+  (a+b)*2
 end
 
-Diagnostic.defined_method = # call your method here
+Diagnostic.defined_method = mutiply_sum_by_2
 
```

Comment:
looking for something like this: Diagnostic.defined_method = add_and_multiply(3, 4)